lillian_klasen/login_reg/server.py

```diff
@@ -0,0 +1,96 @@
+from flask import Flask, render_template, request, redirect, flash, session
+from mysqlconnection import MySQLConnector
+import re
+import md5
+
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+app = Flask(__name__)
+app.secret_key = "secret"
+mysql = MySQLConnector(app, 'logins')
+
+@app.route('/')
+def index():
+  return render_template('index.html')
+
+
+@app.route('/process', methods=['POST'])
+def process():
+    if len(request.form['first_name']) < 2:
+        flash('Please enter a first name!')
+        return redirect('/')
+
+    elif request.form['first_name'].isalpha() == False:
+        flash("You cannot have any numbers in your first name")
+        return redirect('/')
+
+    elif request.form['last_name'].isalpha() == False:
+        flash("You cannot have any numbers in your last name")
+        return redirect('/')
+
+    elif len(request.form['last_name']) <2:
+        flash("Please enter a last name")
+        return redirect('/')
+
+    elif not EMAIL_REGEX.match(request.form['email']):
+        flash("Please enter a valid email address")
+        return redirect('/')
+
+    elif len(request.form['password']) < 8:
+        flash("Please enter a valid password")
+        return redirect('/')
+
+    elif len(request.form['confirm_password']) < 1:
+        flash("Please confirm your password")
+        return redirect('/')
+
+    elif request.form['confirm_password'] != request.form['password']:
+        flash("Passwords do not match")
+        return redirect('/')
+
+    else:
```

Comment:
This now makes sense for why you have elif statements. I would suggest not repeating yourself in your code so much. For instance, you currently have `return redirect('/')` typed over and over again which is not great practice. Try to optimize this method so that you're not repeating yourself so much.